src/views/common/Login.vue

```diff
@@ -7,38 +7,42 @@
 
                 <div class="w-full lg:w-3/5 mx-4 flex-auto items-center justify-center">
                     <div class="mb-6 text-center">
-                        <i class="m-3 fas fa-envelope absolute text-gray-500"></i>
+                        <i class="mt-4 m-3 ml-4 fas fa-envelope absolute text-gray-500"></i>
                         <input
                             id="email"
                             v-model="email"
-                            class="inline-block center lg:w-3/4 block appearance-none shadow-md font-semibold bg-gray-200 text-gray-600 placeholder-gray-600 focus:text-gray-600 p-2 pl-10 rounded hover:border-gray-300 focus:outline-none focus:shadow-outline"
+                            class="inline-block center lg:w-3/4 block font-semibold p-2 pl-10 form-input input-text"
                             type="text"
                             placeholder="Email"
+                            :class="{ error: error }"
+                            @change="hideErrors()"
                         />
                     </div>
 
                     <div class="mb-6 text-center">
-                        <i class="m-3 fas fa-lock absolute text-gray-500"></i>
+                        <i class="mt-4 m-3 ml-4 fas fa-lock absolute text-gray-500"></i>
                         <input
                             id="password"
                             v-model="password"
                             :type="passwordFieldType"
-                            class="lg:w-3/4 shadow-md inline-block center appearance-none font-semibold bg-gray-200 text-gray-600 placeholder-gray-600 focus:text-gray-600 p-2 pl-10 rounded hover:border-gray-300 focus:outline-none focus:shadow-outline"
+                            class="lg:w-3/4 inline-block center appearance-none font-semibold p-2 pl-10 form-input input-text"
                             placeholder="Password"
-                            @input="hideErrors()"
+                            :class="{ error: error }"
+                            @change="hideErrors()"
                         />
                         <button
                             id="togglePassword"
                             type="button"
                             tabIndex="-1"
-                            class="display-none absolute inline-block center ml-3 visible text-gray-500 text-sm hover:text-gray-600 focus:outline-none"
+                            class="display-none absolute inline-block center ml-3 mt-2 visible text-gray-500 text-sm hover:text-gray-600 focus:outline-none"
```

Comment:
```suggestion
                            class="display-none absolute inline-block center ml-5 mt-2 visible text-gray-500 text-sm hover:text-gray-600 focus:outline-none"
```
The margin to the left was too little.
Before:
![before](https://user-images.githubusercontent.com/63233799/88534747-06778280-d009-11ea-9651-b301b83eb937.PNG)

After:
![after](https://user-images.githubusercontent.com/63233799/88534776-10998100-d009-11ea-9217-cf2b1df50397.PNG)